internal/provider/kubernetes/controller.go

```diff
@@ -1067,6 +1067,11 @@ func (r *gatewayAPIReconciler) processSecurityPolicies(
 	}
 
 	for _, policy := range securityPolicies.Items {
+		if err := validation.ValidateSecurityPolicy(&policy); err != nil {
+			r.log.Error(err, "failed to process SecurityPolicy", "namespace", policy.Namespace, "name", policy.Name)
```

Comment:
we dont want to run this here, we want to save this, run validation and attach the errors in the status (which happen in the next gateway api layer) and then flag that to the user

Reply:
You meant like this? 0e44d8258a369de382f0371fb0698a679fc2beb8

Reply:
yes, thanks !